test/jdk/java/foreign/callarranger/TestWindowsCallArranger.java

```diff
@@ -25,6 +25,7 @@
 /*
  * @test
  * @enablePreview
+ * @requires ((os.arch == "amd64" | os.arch == "x86_64") & sun.arch.data.model == "64") | os.arch == "aarch64"
```

Comment:
Some of the other call arranger tests just use `@requires sun.arch.data.model == "64"`. Please do that here as well. It should be fine to run this test as long as the platform is 64 bit.
```suggestion
 * @requires sun.arch.data.model == "64"
```

Reply:
FWIW, the underlying issue with this test is that the size of the address layout changes across platforms.